Sources/SwiftBAMDC/SwiftMetricsBAMConfig.swift

```diff
@@ -570,26 +572,27 @@ public class BMConfig : IBAMConfig {
                         //var result: String = NSString (data: receivedData, encoding: String.Encoding.utf8.rawValue)
                         let json = try? JSONSerialization.jsonObject(with: receivedData, options: [])
                         
-                        switch (httpResponse.statusCode) {
+                        guard let httpStatus = httpResponse as? HTTPURLResponse else { return }
```

Comment:
Why is this called `httpStatus`? We're checking that it's an `HTTPURLResponse` so `httpURLResponse` would be the correct name for it.

Reply:
Add a log message in the failure case.

Reply:
Shouldn't we call `taskCallback()` here?